app/i18n/en-us/admin.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+return array(
+	'auth' => array(
+		'allow_anonymous' => 'Allow anonymous reading of the default user\'s articles (%s)',
+		'allow_anonymous_refresh' => 'Allow anonymous refresh of the articles',
+		'api_enabled' => 'Allow <abbr>API</abbr> access <small>(required for mobile apps)</small>',
+		'form' => 'Web form (traditional, requires JavaScript)',
+		'http' => 'HTTP (for advanced users with HTTPS)',
+		'none' => 'None (dangerous)',
+		'title' => 'Authentication',
+		'title_reset' => 'Authentication reset',
+		'token' => 'Authentication token',
+		'token_help' => 'Allows access to RSS output of the default user without authentication:',
+		'type' => 'Authentication method',
+		'unsafe_autologin' => 'Allow unsafe automatic login using the format:	-> todo',
+	),
+	'check_install' => array(
+		'cache' => array(
+			'nok' => 'Check permissions on <em>./data/cache</em> directory. HTTP server must have write permission.',
+			'ok' => 'Permissions on the cache directory are good.',
+		),
+		'categories' => array(
+			'nok' => 'Category table is improperly configured.',
+			'ok' => 'Category table is okay.',
+		),
+		'connection' => array(
+			'nok' => 'Connection to the database cannot be established.',
+			'ok' => 'Connection to the database is okay.',
+		),
+		'ctype' => array(
+			'nok' => 'Cannot find a required library for character type checking (php-ctype).',
+			'ok' => 'You have the required library for character type checking (ctype).',
+		),
+		'curl' => array(
+			'nok' => 'Cannot find the cURL library (php-curl package).',
+			'ok' => 'You have the cURL library.',
+		),
+		'data' => array(
+			'nok' => 'Check permissions on <em>./data</em> directory. HTTP server must have write permission.',
+			'ok' => 'Permissions on the data directory are good.',
+		),
+		'database' => 'Database installation',
+		'dom' => array(
+			'nok' => 'Cannot find a required library to browse the DOM (php-xml package).',
+			'ok' => 'You have the required library to browse the DOM.',
+		),
+		'entries' => array(
+			'nok' => 'Entry table is improperly configured.',
+			'ok' => 'Entry table is okay.',
+		),
+		'favicons' => array(
+			'nok' => 'Check permissions on <em>./data/favicons</em> directory. HTTP server must have write permission.',
+			'ok' => 'Permissions on the favicons directory are good.',
+		),
+		'feeds' => array(
+			'nok' => 'Feed table is improperly configured.',
+			'ok' => 'Feed table is okay.',
+		),
+		'fileinfo' => array(
+			'nok' => 'Cannot find the PHP fileinfo library (fileinfo package).',
+			'ok' => 'You have the fileinfo library.',
+		),
+		'files' => 'File installation',
+		'json' => array(
+			'nok' => 'Cannot find JSON (php-json package).',
+			'ok' => 'You have the JSON extension.',
+		),
+		'mbstring' => array(
+			'nok' => 'Cannot find the recommended library mbstring for Unicode.',
```

Comment:
You forgot to change this line, but I think it's probably slightly more elegant to fix up the Franglais by simply reversing the order. In English it's `mbstring library`, in French `librairie mbstring`.
```suggestion
			'nok' => 'Cannot find the recommended mbstring library for Unicode.',
```

Reply:
Agree, `mbstring library` sounds cleaner.  will revise